data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -93,20 +93,7 @@
     "required_str": 16,
     "looks_like": "f_washer",
     "flags": [ "CONTAINER", "PLACE_ITEM", "BLOCKSDOOR", "FLAT_SURF" ],
-    "deconstruct": {
```

Comment:
That doesn't make sense, the furniture autoclave is a big machine at least the size of a washing machine, it can't just turn into a microwave size autoclave.

Reply:
> None of those places have power, autoclave furnitures only make sense in labs because we assume the labs still have electricity.

Are we assuming that all furniture in CDDA was placed after the cataclysm by some unknown force that was cognizant of where the power supply still worked, or was the furniture already there from pre-cataclysm? /s

> That doesn't make sense, the furniture autoclave is a big machine at least the size of a washing machine, it can't just turn into a microwave size autoclave.

This results in the autoclave - which is 100% necessary for installing CBMs - being nigh unattainable. I didn't modify the "autoclave" item, it has always existed as it does now. Though yes, the autoclave item is smaller than it should be for an autoclave, it is still over double the volume of the average microwave.

Deconstruction into the item simply simulates removing it from the existing power / plumbing infrastructure. The item should represent the same size as the original furniture, simply portable so it can be reinstalled as a vehicle autoclave. The item itself is (iirc) pretty much unusable without installation since the power required is much larger than any available non-plutonium battery.

Reply:
> we assuming that all furniture in CDDA was placed after the cataclysm by some unknown force that was cognizant of where the power supply still worked, or was the furniture already there from pre-cataclysm? /s

The furniture doesn't use any power and just use water, currently we only simulate power on the map by placing all the stuff that works ( autotdoc, lights, autocalves..) in labs where we pretend there is power. If you put the autoclave in a doctor office where there shouldn't be any power it will work anyway despite the fact that it shouldn't.

> Deconstruction into the item simply simulates removing it from the existing power / plumbing infrastructure. The item should represent the same size as the original furniture, simply portable so it can be reinstalled as a vehicle autoclave.

Just add the autoclave tool as is in those places and leave the deconstruction recipe of the furniture autoclave as it is. It's super weird to just transform a huge standing autoclave into a smaller version.